test/LCS Program/LCS.py

```diff
@@ -0,0 +1,25 @@
+# Dynamic Programming implementation of LCS problem by Jasani Dhruv
```

Comment:
I believe this program already exists in the Python folder. This one seems out of place (in "test").